_posts/2018-02-13-jennifer-isasi-jose-a-motilla.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Jennifer Isasi y José Antonio Motilla se unen al equipo de The Programming Historian en español 
+authors:
+- Maria José Afanador-Llach
+layout: post
+categories: posts
+---
+
+<p><figure><img src="/avatars/Jennifer-Isasi.png" alt="" style="width:200px;"/><img src="/avatars/Jose-Antonio-Motilla.png" alt="" style="width:200px;"/><figcaption>
+    Jennifer Isasi y José Antonio Motillo se han unido a The Programming Historian</figcaption></figure></p>
+    
+¡The Programming Historian tiene el placer de presentar sus dos nuevos miembros del equipo de editores de contenidos en español!
+
+Continuando con nuestro compromiso con la diversidad lingüística para hacer los tutoriales de PH accesibles al mundo hispanoparlante y seguir construyendo una comunidad de colaboradores, en noviembre de 2017 el comité editorial de PH anunció una convocatoria para editor de contenidos en español. Recibimos aplicaciones excelentes y decidimos invitar a los dos finalistas para la posición.  
+
+Nos emociona anunciar que Jennifer Isasi y José Antonio Motilla han aceptado la invitación. Sin más preámbulos, les presentamos a los nuevos miembros del comité editorial de PH:
+
+[Jennifer Isasi](@jenniferisve) obtuvo recientemente su doctorado en Estudios Hispánicos de la Universidad de Nebraske-Lincoln. Sus intereses de investigación son la literatura, cultura e historia española de los siglos XVI al XXI, las humanidades digitales, los estudios culturales y el impacto de los nuevos medios y tecnologías de la información sobre las culturas hispánicas. Tiene dos certificados en Humanidades Digitales y experiencia en el desarrollo de metodologías de extracción, análisis y visualización de datos. Le interesa afrontar el reto de la falta de herramientas adaptadas al español para enseñanza e investigación. 
```

Comment:
Are you trying to link to Jennifer's and José's GitHub pages? If so, you need to put the full URL here, not just the username.

Reply:
@mariajoafana you need to add the URLs to the markdown file in a new commit on this branch.

Reply:
I was trying to link their Twitter accounts